Program.cs

```diff
@@ -63,8 +63,12 @@ class Program {
                 Console.WriteLine(s);
         }
         static ConverterBase CreateConverter(string extension, Dictionary<string, string> argDictionary, string outputPath) {
+
+            Func<string, bool> ExtensionEquals = (extensionCompare)
+                => string.Equals(extension, extensionCompare, StringComparison.CurrentCultureIgnoreCase);

```

Comment:
please, use the 'InvariantCultureIgnoreCase' instead. so the result should not be different on different machines.